botocore/args.py

```diff
@@ -260,10 +260,17 @@ def compute_client_args(
                 tcp_keepalive=client_config.tcp_keepalive,
                 user_agent_extra=client_config.user_agent_extra,
                 user_agent_appid=client_config.user_agent_appid,
+                request_min_compression_size_bytes=(
+                    client_config.request_min_compression_size_bytes
+                ),
+                disable_request_compression=(
+                    client_config.disable_request_compression
+                ),
```

Comment:
For formatting, we could alternatively do the following. The names would need to be very short for it to work with black.
```suggestion
         min_comp_size =  client_config.request_min_compression_size_bytes
         disable_comp =  client_config.disable_request_compression
         ...
                request_min_compression_size_bytes=min_comp_size,
                disable_request_compression=disable_comp,
```